pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string Username = 3;
+    // FriendlyName, if set, is a shorter name for the user than
+    // Username.  Unlike Username it does not identify the user (it
+    // might not be unique); use it in the user's GUI rather than in
+    // backend code.
+    string FriendlyName = 4;
+    string Email = 5;
+    bytes EncryptedPassword = 6;
+}
+
+message GroupData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string DisplayName = 3;
+}
+
+message PolicyData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string DisplayName = 3;
+    StatementsData Statements = 4;
```

Comment:
Are we saving StatmentData separately or as part of other data models? We need to decide how to save this data in the DB

Reply:
it is saved as a part of the policy model, but because it is not a regular type we need to convert it too